vdbench_wrapper/vdbench_wrapper.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import os
+import logging
+import requests
+import tarfile
+
+from vdbench_wrapper import trigger_vdbench
+
+logger = logging.getLogger('snafu')
+
+BASE_PATH = '/opt/vdbench'
+
+
+class vdbench_wrapper():
+
+    def __init__(self, parser):
+
+        # collect arguments
+        parser.add_argument(
+            '-c', '--config',
+            help='Provide configuration file')
+        parser.add_argument(
+            '-o', '--output',
+            help='path to output files',
+            default=BASE_PATH+'/outputs')
+        parser.add_argument(
+            '-s', '--sample',
+            help='number of times to run benchmark, defaults to 1',
+            type=int,
+            default=1)
+        parser.add_argument(
+            '-l', '--log',
+            help='log file (Full path)',
+            default=BASE_PATH+'/logs')
+        parser.add_argument(
+            '-d', '--dir',
```

Comment:
why do you need this if you have output dir above?

Reply:
The purpose of the directorys is different.
The first one (output dir) is used for the benchmark output itself for each test. and this one is where to put the Results.tgz file which contain all outputs, logs and reports from all samples.